### ISSUE TITLE
Must .vault-stats in the root of obsidian folder?

### ISSUE THREAD

**Issue by linfz94** [lukeleppan/better-word-count#18]
Hi there, I would like to pose an enhance. I wonder whether the .vault-stats file exists in the plugin folder rather than the Obsidian root folder.

**Comment by lukeleppan** [comment]
Hi. So when I made the decision to put the .vault-stats folder in the vault's root, I wanted it to work with obsidian's sync. At the time it didn't sync plug-in settings. I haven't tested to see if it syncs everything in the .obsidian folder. Is there a specific reason as to why it should be moved to the .obsidian folder?

**Comment by lukeleppan** [comment]
Yeah so with the latest update it stores the file in the vault-stats.json in the .obsidian folder. But I just remember that I haven't tested that so maybe it doesn't work with sync I guess for now we can only hope.
